test/factories/docker/run.spec.js

```diff
@@ -20,6 +20,17 @@ describe('Docker Run Command', () => {
     },
     {
       options: {
+        detach: true,
+        target: {
+          image: 'usher',
+          tag: 'test',
+          registry: 'myReg'
+        }
+      },
+      expected: "docker run -d myReg/usher:test"
```

Comment:
We're currently re-asserting the "myReg/usher:test" format with every test. Is there any way to do this differently so the test only fails if we miss the -d option as that's the only thing we really care about in this test.